Wikimate.php

```diff
@@ -1432,6 +1434,10 @@ public function delete($reason = null)
         if (!is_null($reason)) {
             $data['reason'] = $reason;
         }
+        // Check if deletetalk parameter is supported (it is since MediaWiki v1.38)
```

Comment:
```suggestion
        // Check if the `deletetalk` parameter is supported (it is since MediaWiki v1.38)
```

(Does markdown code formatting work in the rendered docs?)

Reply:
Uhm, inline comments are not included in the rendered docs, so the backticks are not useful. And to keep the line length close to 80 characters (for undeletetalk in the next PR too), I think it's fine to leave out "the" here. It's just a comment after all. :)

Reply:
Uh, silly me, I clearly wasn't paying attention. Ok, fair enough, let's disregard this suggestion.